registry/keda-scaler.yaml

```diff
@@ -0,0 +1,41 @@
+# Code generated by KubeVela templates. DO NOT EDIT.
+apiVersion: core.oam.dev/v1beta1
+kind: TraitDefinition
+metadata:
+  name: keda-scaler
+  annotations:
+    definition.oam.dev/description: "keda-scaler supports multiple event to elastically scale applications, this scaler only applies to ScaledObject"
+spec:
+  definitionRef:
+    name: scaledobjects.keda.sh
+  appliesToWorkloads:
+    - deployments.apps
+  schematic:
+    cue:
+      template: |
+        outputs: kedaScaler: {
+        	apiVersion: "keda.sh/v1alpha1"
+        	kind:       "ScaledObject"
+        	metadata: {
+        		name: context.name
+        	}
+        	spec: {
+        		scaleTargetRef: {
+        			name: context.name
```

Comment:
No namespace?

Reply:
The namespace field is not included here.

https://github.com/kedacore/keda/blob/1f48e663f4c053a3ff3d7d01c4a82f97b5147141/api/v1alpha1/scaledobject_types.go#L64